src/consumers.ts

```diff
@@ -85,8 +89,16 @@ class AnalysisConsumer implements IConsumer
 {
     binding: IBindingDescriptor;
     changeToBinding: IBindingDescriptor;
+    regLinkBinding : IBindingDescriptor;
+    messageBinding : IBindingDescriptor;
+    pubVulBinding : IBindingDescriptor;
+    pvtVulBinding : IBindingDescriptor;
     item: any;
     changeTo: string = null;
+    regLink: string = null;
```

Comment:
Use verbose naming. e.g `privateVulnerability` instead of `pvtVul`

Reply:
+1